packages/amplify-category-auth/src/provider-utils/awscloudformation/service-walkthrough-types.ts

```diff
@@ -98,6 +98,7 @@ export type PasswordPolicy = 'Requires Lowercase' | 'Requires Numbers' | 'Requir
 
 export type UsernameAttributes = 'email' | 'phone_number';
 
+export type AliasAttributes = 'email' | 'phone_number' | 'preferred_username';
```

Comment:
could you please use constants here instead of strings
 e.g ATTRIBUTE_TYPE.EMAIL/PHONE/PREFERRED_USERNAME